### PR TITLE
Disable kdump for %arm - boo#1203888

### PR DESCRIPTION
**Pull request by ggardet**

### CODE REVIEW

**Review by ancorgs — APPROVED** [review]
LGTM

**Comment by teclator** [comment]
I guess we should ensure these changes do not collide with the current proposal as kdump is enabled by default 

- see https://github.com/yast/skelcd-control-MicroOS/blob/master/control/control.MicroOS.xml#L73

**Comment by ggardet** [comment]
> I guess we should ensure these changes do not collide with the current proposal as kdump is enabled by default
> 
> * see https://github.com/yast/skelcd-control-MicroOS/blob/master/control/control.MicroOS.xml#L73

Not sure how to add a condition on the architecture in this file. Any suggestion?

**Comment by ancorgs** [comment]
> I guess we should ensure these changes do not collide with the current proposal as kdump is enabled by default

I would certainly expect that to be a problem. I overlooked it in my LGTM. Sorry.

To be honest, I assumed this was manually tested before submitting it (I should have asked). 

The bad news is that I merged the corresponding PR for Factory (https://github.com/yast/skelcd-control-openSUSE/pull/256). Although maybe that is not a problem since I think kdump is not part of the installer proposal in Tumbleweed.

**Comment by ggardet** [comment]
> > I guess we should ensure these changes do not collide with the current proposal as kdump is enabled by default
> 
> I would certainly expect that to be a problem. I overlooked it in my LGTM. Sorry.
> 
> To be honest, I assumed this was manually tested before submitting it (I should have asked).
> 
> The bad news is that I merged the corresponding PR for Factory ([yast/skelcd-control-openSUSE#256](https://github.com/yast/skelcd-control-openSUSE/pull/256)). Although maybe that is not a problem since I think kdump is not part of the installer proposal in Tumbleweed.

That's good news, it will be a good test. This is currently broken anyway, so not a big deal.

**Review by teclator — CHANGES_REQUESTED** [review]
Please add version bump

File: package/skelcd-control-MicroOS.changes [hunk 1]
```diff
@@ -1,3 +1,8 @@
+-------------------------------------------------------------------
+Thu Sep 29 12:52:01 UTC 2022 - Guillaume GARDET <guillaume.gardet@opensuse.org>
+
+- Disable kdump for %arm - boo#1203888
```
Comment (teclator) [inline package/skelcd-control-MicroOS.changes:4]:
```suggestion
- Disable kdump for %arm - boo#1203888
- 20221010
```

**Review by teclator — APPROVED** [review]

**Pull request merged**